examples/tutorial.pol

```diff
@@ -1,59 +1,103 @@
+-- We can define the data type Bool with constructors T and F.
 data Bool { T, F }
 
+-- We can define the not operator by case distinction.
```

Comment:
```suggestion
-- We can define `not`, which negates a boolean value, by case distinction.
```
We have to come up with a consistent naming for what we call things like "not" here. I would reserve "operator" for the syntactic category of prefix, postfix and infix operators.

Reply:
I would call them function in a functional language but of course it's not really that ~ maybe method? but that would be ignorant as well.

Reply:
The internal name is "definition", but that is not really inspired either. But we can solve that in the future :)